### DISCUSSION TITLE
> you do not have enough IDs available in the rootless user namespace.

### DISCUSSION THREAD

**Discussion by hmntsharma** [containers/podman#21138] [General]
> you do not have enough IDs available in the rootless user namespace.
> 
> You need to allocate more IDs to your user.
> 
> Please take a look at the subuid(5) man page for more information on how to do it.

Thanks, I thought that is required when you actually run the container and here I  struggling to even load the image.

How is this not a problem when pulled from a public repository but doesn't work when the same image is archived and loaded on another machine?

_Originally posted by @hmntsharma in https://github.com/containers/podman/issues/21037#issuecomment-1867819987_

**Comment by hmntsharma** [comment]
Kindly help me with an example of the solution. I haven't had any luck so far. Thanks!

**Comment by rhatdan** [comment]
Most likely the account on the second machine is not setup with enough UIDs/GIDs defined in its /etc/subuid and /etc/subgid files.

**Comment by hmntsharma** [comment]
Since the two machines have distinct users, assigning the second machine's user the same range of sub UIDs as the first machine resolved the issue.

[Fix found here](https://docs.podman.io/en/latest/markdown/podman.1.html#rootless-mode)
